### PR TITLE
[fix bug 1335569] Add health subpage - Digital Inclusion

### PR DESCRIPTION
**Pull request by craigcook**
## Description
Adds the new digital inclusion page.

Also redesigns the existing privacy and security page. String changes are behind the tag `internet_health_digital_inclusion`

This also updates some text on the internet health landing page. String changes there are behind the same tag, `internet_health_digital_inclusion`

## Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1335569

## Testing
https://bedrock-demo-craigcook.us-west.moz.works/en-US/internet-health/digital-inclusion/

## Checklist
- [x] Requires l10n changes.
- [ ] Related functional & integration tests passing.

### CODE REVIEW

File: media/css/mozorg/internet-health/health-subpage.scss [hunk 1]
```diff
@@ -0,0 +1,747 @@
...
+            &:first-child {
+                border-right: none;
+            }
+        }
+    }
+}
+
+
+//*--------------------------------------------------------------------------*/
+// Health subpages navigation
```
Comment (craigcook) [inline media/css/mozorg/internet-health/health-subpage.scss]: We'll need a better plan for what the navbar looks like on mobile. For now this is good enough with only two links, but once we add three and four it's gonna get clunky.

**Comment by alexgibson** [comment]
Is the left/right lack margin either side of the mobile nav items intentional?

mobile-nav

File: bedrock/mozorg/redirects.py [hunk 2]
```diff
@@ -646,6 +646,9 @@ def decider(request, **kwargs):
     # Bug 1322959 - vanity URL
     redirect(r'^onlineprivacy/?$', 'mozorg.internet-health.privacy-security'),
 
+    # Bug 1335569 - vanity URL
+    redirect(r'^digital-inclusion/?$', 'mozorg.internet-health.digital-inclusion'),
```
Comment (alexgibson) [inline bedrock/mozorg/redirects.py]: Would be nice to have a test for this vanity redirect? (one above also?)

**Comment by alexgibson** [comment]
Newsletter thank you text overlaps the background image:

newsletter

File: bedrock/mozorg/templates/mozorg/internet-health/digital-inclusion.html [hunk 3]
```diff
@@ -0,0 +1,413 @@
...
+
+            <p class="note-source">
+              <cite>
+              {% trans wwwf='http://webfoundation.org/about/research/womens-rights-online-2015/' %}
+                Source: <a href="{{ wwwf }}">World Wide Web Foundation</a>, 2015
+              {% endtrans %}
+              </cite>
+            </p>
+          </div>
+          <span class="aside-accent" aria-hidden="true" role="presentation">50%</span>
```
Comment (craigcook) [inline bedrock/mozorg/templates/mozorg/internet-health/digital-inclusion.html]: It certainly could, but then it needs an ID or unique class for each box so they can have different content in the CSS. Trying to plan ahead for more of these pages, this seemed a bit DRYer and the lesser evil.

I also thought there may be the need for these to be localizable in some cases, depending on the content. Right now they're all numbers so we don't need to stringify them (probably?), though I'm now having second thoughts about "1B" on the privacy page. Maybe they do need to be strings after all. I'm not sure about "x%" in other languages now that I think about it.

Alright, I've convinced myself. I'll stringify them.

File: bedrock/mozorg/templates/mozorg/internet-health/digital-inclusion.html [hunk 4]
```diff
@@ -0,0 +1,413 @@
...
+                Supporting the <a href="{{ coral }}">Coral Project</a>, which works to improve how journalists and
+                communities engage on news websites, including tools like <a href="{{ talk }}">Talk</a>, that aims
+                to create smarter, safer comments sections.
+              {% endtrans %}
+              </p>
+            </li>
+            <li>
+              <h4>{{ _('Empowering individuals') }}</h4>
+              <p>
+              {% trans teaching='https://d157rqmxrxj6ey.cloudfront.net/amira/33977/' %}
```
Comment (craigcook) [inline bedrock/mozorg/templates/mozorg/internet-health/digital-inclusion.html]: Looks like it also lives at https://thimbleprojects.org/amira/33977/, which is much nicer. I'll update.

**Comment by craigcook** [comment]
I always forget the thank you text... I really hope we can stop redesigning this form on every new page soon.

**Comment by alexgibson** [comment]
> I really hope we can stop redesigning this form on every new page soon.

Yeah, something I'd very much like us to make a component in the truest sense +1

File: bedrock/mozorg/templates/mozorg/internet-health.html [hunk 5]
```diff
@@ -81,6 +81,17 @@
     <section id="innovation" class="even">
       <div class="content">
         <div class="inner-container">
+        {% if l10n_has_tag('internet_health_digital_inclusion') %}
```
Comment (flodolo) [inline bedrock/mozorg/templates/mozorg/internet-health.html]: Note for @peiying2: Ignore this for now, we already have this text in the file (I guess wrong copy and paste)

Comment (craigcook) [inline bedrock/mozorg/templates/mozorg/internet-health.html]: Yes, apologies, this isn't new text after all. It was fixed in ee5105c25b063506875a086a49203deddb72d411.

File: bedrock/mozorg/templates/mozorg/internet-health/privacy-security.html [hunk 6]
```diff
@@ -77,90 +97,119 @@ <h2 class="topic-title">{{ _('User Control: Deciding who can collect your data')
...
+          <ul>
             <li>
-            {% trans
-                ios='https://support.apple.com/HT203033',
-                android='https://support.google.com/googleplay/answer/6014972' %}
-              Make sure your mobile apps access only the info they need. Control your
-              privacy and location settings on <a href="{{ ios }}" rel="external">iOS</a>
-              and <a href="{{ android }}" rel="external">Android</a>.
-            {% endtrans %}
+            {% if l10n_has_tag('internet_health_subpage_rebrand') %}
```
Comment (flodolo) [inline bedrock/mozorg/templates/mozorg/internet-health/privacy-security.html]: Next time, let's not use two different l10n tags in the same update.

File: bedrock/mozorg/templates/mozorg/internet-health/privacy-security.html [hunk 7]
```diff
@@ -352,9 +451,11 @@ <h3 class="topic-action-title">{{ _('What Mozilla is doing about government surv
     <div class="content">
       <h2>{{ _('Hungry for more?') }}</h2>
       <p class="report-cta">
-      {% trans healthreport='https://internethealthreport.org' %}
-        Keep reading about Privacy and Security in our <a href="{{ healthreport }}">Internet Health Report</a>.
-      {% endtrans %}
+        <span>
+        {% trans healthreport='https://internethealthreport.org' %}
+          Read on about Digital Inclusion in our <a href="{{ healthreport }}">Internet Health Report</a>.
```
Comment (flodolo) [inline bedrock/mozorg/templates/mozorg/internet-health/privacy-security.html]: This change should have been behind a tag.

**Pull request merged**